dev/ci/ci-sf.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 
 ci_dir="$(dirname "$0")"
-source ${ci_dir}/ci-common.sh
+. "${ci_dir}/ci-common.sh"
 
-mkdir -p ${CI_BUILD_DIR} && cd ${CI_BUILD_DIR}
-wget -qO- ${sf_lf_CI_TARURL}  | tar xvz
-wget -qO- ${sf_plf_CI_TARURL} | tar xvz
-wget -qO- ${sf_vfa_CI_TARURL} | tar xvz
+mkdir -p "${CI_BUILD_DIR}" && cd "${CI_BUILD_DIR}" || exit
```

Comment:
This `|| exit` shouldn't be necessary considering we just did `mkdir -p`.
If you still want to silence shellcheck it should be `exit 1`.